cherrypy/lib/static.py

```diff
@@ -75,6 +75,13 @@ def serve_file(path, content_type=None, disposition=None, name=None,
         i = path.rfind('.')
         if i != -1:
             ext = path[i:].lower()
+            if ext in mimetypes.encodings_map.keys():
```

Comment:
Also I'm not sure, but probably this behavior must be configurable.
Let's ask @jaraco's opinion.